phylogenetic/config/auspice_config.json

```
@@ -1,36 +1,45 @@
-{
-  "title": "Real-time tracking of Lassa virus evolution",
-  "color_options": {
-    "gt": {
-      "menuItem": "genotype",
-      "legendTitle": "Genotype",
-      "type": "discrete",
-      "key": "genotype"
-    },
-    "num_date": {
-      "menuItem": "date",
-      "legendTitle": "Sampling date",
-      "type": "continuous",
-      "key": "num_date"
-    },
-    "authors": {
-      "key":"authors",
-      "legendTitle":"Authors",
-      "menuItem":"authors",
-      "type":"discrete"
-    },
-    "country": {
-    }
-  },
-  "geo": [
-    "country"
-  ],
-  "maintainer": [
-    "ACEGID Redeemer's University",
-    "http://acegid.org/"
-  ],
-  "filters": [
-    "country",
-    "authors"
-  ]
-}
+{
+  "title": "Real-time tracking of Lassa virus evolution",
+  "data_provenance": [
+    {
+      "name": "GenBank",
+      "url": "https://www.ncbi.nlm.nih.gov/genbank/"
+    }
+  ],
+  "colorings": [
+    {
+      "key": "gt",
+      "title": "Genotype",
+      "type": "categorical"
+    },
+    {
+      "key": "num_date",
+      "title": "Date",
+      "type": "continuous"
+    },
+    {
+      "key": "author",
+      "title": "Author",
+      "type": "categorical"
+    },
+    {
+      "key": "country",
+      "title": "Country",
+      "type": "categorical"
+    }
+  ],
+  "geo_resolutions": [
+    "country"
+  ],
+  "maintainer": [
+    "ACEGID Redeemer's University",
+    "http://acegid.org/"
+  ],
+  "filters": [
+    "country",
+    "authors"
```

```suggestion
    "author"
```

Noticed because `augur export` was outputting a warning
```
WARNING:  The filter "authors" does not appear as a property on any tree nodes.
```